omf/fileio/geoh5.py

```diff
@@ -647,18 +647,23 @@ def collect_omf_attributes(element: MappedData, **kwargs) -> dict:
         if not element.legends:
             return kwargs
 
-        value_map = {
-            count + 1: str(val) for count, val in enumerate(element.legends[0].values)
-        }
-        color_map = np.vstack(
-            [
-                np.r_[count + 1, val, 1.0]
-                for count, val in enumerate(element.legends[1].values)
-            ]
-        )
+        value_map = {}
+        color_map = [np.r_[0, 0, 0, 0, 0]]
```

Comment:
Just a question: 
we are appending a new count for every. Why is the first raw full of zeros?
For unknown?
Is unknown always zero? Or could it be something else?
Heard JP already have some bug about unknown being not zeros

Reply:
That's what this issue is addressing. First value is in the index (reserved for GA 0=="Unknwow". The color doesn't really matter [0, 0, 0] is black, and the alpha is 0 (transparent).